.env.defaults

```diff
@@ -52,8 +52,8 @@ YRBL_HAPPY_IMG_URL=https://spee.ch/spaceman-happy:a.png?quality=85&height=457&wi
 YRBL_SAD_IMG_URL=https://spee.ch/spaceman-sad:d.png?quality=85&height=457&width=457
```

Comment:
Overlap between Settings and Balance

- Split screen 50-50 with another app, 1920x1080 screen (common)
- Full screen + 1080p monitor in portrait (not common)

![image](https://user-images.githubusercontent.com/64950861/146696525-da00b54f-546b-4ad7-aa81-2c018ec366a6.png)